webui/react/src/types.ts

```diff
@@ -184,7 +184,7 @@ interface ExperimentHyperParam {
   maxval?: number;
   minval?: number;
   type: string;
-  val?: number | string;
+  val?: unknown;
```

Comment:
Does this include Nonetype? i.e. 

hp_var: None

Reply:
Same with 
Categorical:
 types: ['a', None]

Reply:
I think they'd come through as strings since `None` is not a special value in YAML. We seem to have some samples

Reply:
https://github.com/determined-ai/determined/pull/1763/files#diff-98b216143056cb9a93b0e5179a1b513607c33cdcd4052fb2b2aec880c370bff3R1125

ahh we don't have `types: ['a', None]` I think

Reply:
let me clarify as I was shorthanding it a bit: 

HP_var1:
   type: categorical
   vars: ['a', None]

Reply:
I'm confused about the None usage here. 

I'm assuming https://github.com/determined-ai/determined/pull/1763#discussion_r554128598 
translates to json like so: 

```json
  {
    "myParam": {
      "type": "categorical",
      "vals": ["a", "None"]
    }
  },
￼

Reply:
I added a test case with None https://github.com/determined-ai/determined/pull/1763/commits/4bda23bec2578b2897bb3d8145b47fe3fa412419


I just checked to make sure `vars: ['a', None]` the quotations, or lack there of,  don't play a role here.

![Screen Capture_select-area_20210108112556](https://user-images.githubusercontent.com/12127420/104055843-4d66c800-51a4-11eb-940f-c2d55ad40e2c.png)